src/current/_includes/v24.1/misc/session-vars.md

```diff
@@ -1,6 +1,7 @@
 | Variable name | Description | Initial value | Modify with [`SET`]({% link {{ page.version.version }}/set-vars.md %})? | View with [`SHOW`]({% link {{ page.version.version }}/show-vars.md %})? |
 |---|---|---|---|---|
 | <a id="application-name"></a> `application_name` | The current application name for statistics collection. | Empty string, or `cockroach` for sessions from the [built-in SQL client]({% link {{ page.version.version }}/cockroach-sql.md %}). | Yes | Yes |
+| <a id="autocommit-before-ddl"></a> `autocommit_before_ddl` | {% include {{page.version.version}}/sql/auto-commit-before-ddl.md %}  | `off` | Yes | Yes |
```

Comment:
The include file is throwing off the formatting by putting the next 3 columns in a new row:

<img width="1225" alt="Screenshot 2024-03-26 at 11 23 07 AM" src="https://github.com/cockroachdb/docs/assets/58752716/c5e14e77-f62a-4ae7-a421-151391e0df84">

Maybe try removing white space with a hyphen in the liquid include like `-%}`.

Reply:
oh wow I did not know about removing whitespace with a hyphen, thanks! trying that now

Reply:
From my experience with putting include files in a markdown table removing the whitespace may not work. You may have to use an html table.

Reply:
ok well it didn't work unfortunately!  but at least TIL, so thanks for that!

since the include was used in 2 places, and this one didn't render properly I've reworked in the latest commit to just inline the text in both places and deleted the include file altogether. PTAL